test/kubemark/start-kubemark.sh

```diff
@@ -167,18 +167,7 @@ function create-and-upload-hollow-node-image {
   CURR_DIR=`pwd`
   cd "${MAKE_DIR}"
   RETRIES=3
-  for attempt in $(seq 1 ${RETRIES}); do
```

Comment:
It would be preferable to have this code in this script itself rather than moving it to an interface function in $CLOUD_PROVIDER/util.sh as this step is almost the same for all providers, except for the docker registry location (and possibly the docker push command). You should prefer calling make with the right vars set and these vars should be provided through `test/kubemark/cloud-provider-config.sh`.